data/tutorials/language/1ds_01_arrays.md

```diff
@@ -0,0 +1,158 @@
+---
+id: arrays
+title: Arrays
+description: >
+  The standard library's Array module
+category: "Data Structures"
+---
+
+# Arrays
+
+## Introduction
+
+In OCaml, arrays are collections of elements of the same type that are stored in contiguous memory locations. They are mutable, each value can be updated, but they have a fixed size; which set them apart from lists.
+
+Despite these differences many of the functions readily available on arrays are similar to the ones available for lists, please refer to the [List tutorial](https://ocaml.org/docs/lists) and documentation for more details about these functions.
```

Comment:
```suggestion
Despite these differences, many of the functions readily available on arrays are similar to the ones available for lists. Please refer to the [List tutorial](https://ocaml.org/docs/lists) and documentation for more details about these functions.
```

Reply:
if it is permitted to link to the manual in these docs:

```suggestion
Despite these differences, many of the functions readily available on arrays are similar to the ones available for lists. Please refer to the [List tutorial](https://ocaml.org/docs/lists) and [documentation](https://v2.ocaml.org/api/List.html) for more details about these functions.
```